azure-kusto-data/azure/kusto/data/exceptions.py

```diff
@@ -9,8 +11,8 @@ class KustoError(Exception):
 class KustoServiceError(KustoError):
     """Raised when the Kusto service was unable to process a request."""
 
-    def __init__(self, messages, http_response, kusto_response=None):
-        super(KustoServiceError, self).__init__(self, messages)
+    def __init__(self, messages: Union[str, List[dict]], http_response: requests.Response, kusto_response=None):
```

Comment:
Using types is inconsistent with the rest of the code, I don't want to get into discussion whether its good or not to use types in Python, but I think we should be consistent.

Reply:
I think we should adopt a policy to modify files we touch to include types.
If that's ok with you, I can add types to the rest of the APIs in Exceptions (and any future file I modify).

Reply:
I prefer to keep it without types, we can have a meeting to discuss this.

Reply:
I removed the type hints.
Let's get this pushed and discuss the subject separately